atomix/cluster/src/main/java/io/atomix/raft/protocol/AppendRequest.java

```diff
@@ -17,16 +17,13 @@
 package io.atomix.raft.protocol;
 
 import static com.google.common.base.MoreObjects.toStringHelper;
-import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.base.Preconditions.checkNotNull;
 
 import io.atomix.cluster.MemberId;
-import java.util.Arrays;
 import java.util.List;
 import java.util.Objects;
 
 /**
- * Append entries request.
+ * Append entries request that represent old version (version = 1)
```

Comment:
:wrench: Document what version 1 means, e.g. "Deprecated append request which only replicates the entry's data and not the complete serialized journal record"